### ISSUE TITLE
programm loads, but closes after a few seconds

### ISSUE THREAD

**Issue by mholzy84** [patrickmoore/Mu#15]
programm loads, but closes after a few seconds like its minimized to system tray. But for some reason it stops running. Have tried running it as admin and various compatibility modes

**Comment by MerlinCooper** [comment]
What version are you using? 1.9.3.2?

**Comment by mholzy84** [comment]
Yeah 1.9.3.2. I only run windows defender and even with that off it still behaves the same.

In event viewer the are 2 errors that get logged. One is a error with daq.dll file and the other is a .net runtime error

**Comment by mholzy84** [comment]
Log Name:      Application
Source:        .NET Runtime
Date:          13/06/2020 10:44:11 AM
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      DESKTOP-G2IPVDS
Description:
Application: Mu.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 00007FF86EB3F971
Stack:

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2020-06-13T00:44:11.3581629Z" />
    <EventRecordID>352</EventRecordID>
    <Correlation />
    <Execution ProcessID="0" ThreadID="0" />
    <Channel>Application</Channel>
    <Computer>DESKTOP-G2IPVDS</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application: Mu.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 00007FF86EB3F971
Stack:
</Data>
  </EventData>
</Event>

**Comment by MerlinCooper** [comment]
Did you try to install it not in the C:\Program Files directory?

I'm running the same version w/o no issues. Difference seems that I did not install in the Programes directory tho.

**Comment by mholzy84** [comment]
Yes I did try and install it on my other HDD, but it actually doesn't install in the location I choose. I changed dir to H:\\mu and it still installed in program files. It's rather weird it runs just fine on my laptop also

**Comment by MerlinCooper** [comment]
Have you tried uninstalling, manually deleting remaining files and than re-installing?

**Comment by patrickmoore** [comment]
Install the latest version of the .Net Framework. You currently have 4.0.3 when Mu requires 4.7.2.

**Comment by mholzy84** [comment]
So after checking which versions of .net are installed, I already have the latest version of .net framework installed. 4.8.04084

Perhaps a fresh install of windows is required
